src/main/java/de/dentrassi/rpm/builder/VerifyDetails.java

```diff
@@ -0,0 +1,159 @@
+package de.dentrassi.rpm.builder;
+
+import org.eclipse.packager.rpm.VerifyFlags;
+import org.eclipse.packager.rpm.build.FileInformation;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Each member of this class corresponds to one bit in the verify flags.
+ * See http://ftp.rpm.org/api/4.14.0/group__rpmvf.html.
+ * See https://github.com/ctron/rpm-builder/issues/41
+ */
+public final class VerifyDetails {
+   private Boolean md5;
+   private Boolean size;
+   private Boolean linkto;
+   private Boolean user;
+   private Boolean group;
+   private Boolean mtime;
```

Comment:
I would prefer more "human readable" fields here. As they would be visible in the documentation and POM file I assume.

Reply:
These members and there corresponding bean properties correspond to the fields in the rpmVerifyFlags_e, see http://ftp.rpm.org/api/4.14.0/group__rpmvf.html. They also correspond to the rpm command line switches:
[--nolinkto] [--nofiledigest] [--nosize] [--nouser]
 [--nogroup] [--nomtime] [--nomode] [--nordev]
 [--nocaps]

I have just renamed the member md5 to fileDigest, as the corresponding command line switch --nofiledigest is now prefered against the obsolete --md5.
As far as the other names are concerned, I have no idea how to improve them.